app/components/TrackRow/index.js

```diff
@@ -51,6 +51,12 @@ class TrackRow extends React.Component {
     );
   }
   
+  addToFavorites() {
```

Comment:
The only suggestion I have is to rename this function to something like `canAddToFavorites`, to indicate the intent.

Reply:
Thank you for the suggestion, i have changed the function name to be more specific.